Solutions/Authomize/Analytic Rules/Detect_AWS_IAM_Users.yaml

```diff
@@ -0,0 +1,61 @@
+id: 077eb06a-c011-47f7-8d92-dfc2b1e1d71b
+name: Detect AWS IAM Users
+kind: Scheduled
+description: The policy detects IAM users across your AWS accounts, a practice that should be kept only for a small number of accountsThis is a default definition by Authomize and can be updated using the edit modal.
```

Comment:
Full stop missing
Is it edit model?

Reply:
Yes in this case it is after "edit model" at the end of the sentence.

Reply:
Not updated

Reply:
OK - I think I caught it this time.